leaf-ui/js/object/InputDynamic.js

```diff
@@ -29,6 +29,8 @@ function getDynamics(){
 	    
 	    if (isNaN(parseInt(initValue))){
 			initValue = satValueDict[initValue];		 
+		}else{
```

Comment:
The inital value (in this case marked value) was set as integer while modelspecbuilder class was expecting a string with 4 chars.